docs/linked-esm.jst

```diff
@@ -0,0 +1,107 @@
+<% /*
+  This is an extended version of the resources/parallel/docco.jst that is
+  bundled with Docco 0.8.0.
+  The license of the original file is available over here:
+  https://github.com/jashkenas/docco/blob/master/LICENSE.
+  The extension adds hyperlinking to javascript import statements.
+*/ %>
```

Comment:
I’d be happy to merge a PR over in Docco to add hyperlinking to import statements...

Reply:
More than happy to do so. This comes in two mutually exclusive flavors, though, which is the reason I haven't done this yet:

1. The quick and dirty way, which monkeypatches the hyperlinks into the page post-rendering using jQuery and which only works when the import syntax and path resolution are like in JavaScript. This is what I've done here.
2. A clean but much slower and more difficult way, which should potentially work for all programming languages that Docco supports. This would require not only patching Docco, but also patching highlight.js, since the output of highlight.js currently offers no portable way to detect module references.

Which would you prefer?

Reply:
Ah, I see — in that case, it seems best to leave it as you have it in this PR.